### PR TITLE
(docs:#1203) - Updates the documentation for feedback spa & microservice.

### PR DESCRIPTION
**Pull request by riginoommen**
# Resolves 1203

### 

# Explain the feature
Reviewed and updated the feedback documentation.

## Does this PR introduce a breaking change
No

## Screenshots
N/A

### Ready-for-merge Checklist

- [x] Expected files: all files in this pull request are related to one feature request or issue (no stragglers)?
- [x] Did you update or add any necessary documentation (README.md, WHY.md, etc.)?

### CODE REVIEW

File: packages/documentation-spa/docs/microservices/feedback/feedback-service.md [hunk 1]
```diff
@@ -3,138 +3,233 @@ id: feedback-service
...
 
 ### Component Contributors
 
-1. Anjnee Sharma - [anjsharm@redhat.com](mailto:anjsharm@redhat.com)   <https://github.com/anjneeksharma>
+1. Rigin Oommen - [roommen@redhat.com](mailto:roommen@redhat.com) - [riginoommen (Rigin Oommen) · GitHub](https://github.com/riginoommen)
 
 ## Getting Started
 
-Feedback microservice is accessible from graphql playground with authorization header but in case of production playground is disabled.
+Feedback Microservice is one of the major pillar of one platform ecosystem. This enables the support to share the feedback with the project management tools.
```
Comment (ghanlohar) [inline packages/documentation-spa/docs/microservices/feedback/feedback-service.md:15]:
```suggestion
Feedback Microservice is one of the major pillar of One Platform ecosystem. This enables the support to share the feedback with the project management tools.
```

File: packages/documentation-spa/docs/microservices/feedback/feedback-service.md [hunk 2]
```diff
@@ -3,138 +3,233 @@ id: feedback-service
...
+ * Jira
+ * GitLab
+ * GitHub
 
 ## Usage
 
 ### Introduction
 
-One Platform's server-side Feedback GraphQL Microservice allows users to talk to the feedback database and can perform operations like addfeedback, updatefeedback, deletefeedback and listfeedback.
+Feedback microservice is built using NodeJS. This interface collects the data from graphql API and opens request with project management tools which is mentioned.
```
Comment (ghanlohar) [inline packages/documentation-spa/docs/microservices/feedback/feedback-service.md:26]:
```suggestion
Feedback microservice is built using NodeJS. This interface collects the data from GraphQL API and interacts with the project management tool provided in the configuration.
```

**Review by ghanlohar — APPROVED** [review]

**Review by deshmukhmayur — CHANGES_REQUESTED** [review]
Check the changes requested by @ghanlohar

File: packages/documentation-spa/docs/microservices/feedback/feedback-service.md [hunk 3]
```diff
@@ -3,138 +3,233 @@ id: feedback-service
...
+  }
+}
+```
+@@### List Feedback
+
+- Operation Name - ListFeedback
+- Variables - ```_id```
+
+Example Query
+```bash
```
Comment (hybridx) [inline packages/documentation-spa/docs/microservices/feedback/feedback-service.md:136]:
```suggestion
```graphql
```

File: packages/documentation-spa/docs/microservices/feedback/feedback-service.md [hunk 4]
```diff
@@ -3,138 +3,233 @@ id: feedback-service
...
+### Mutations
+#### Create feedback
+1. Create new feedback to database and open the ticket accordingly.
+
+- Operation Name - CreateFeedback
+
+- Variables - _FeedbackInput_
+
+Example Mutation
+```bash
```
Comment (hybridx) [inline packages/documentation-spa/docs/microservices/feedback/feedback-service.md:179]:
```suggestion
```graphql
```

File: packages/documentation-spa/docs/microservices/feedback/feedback-service.md [hunk 5]
```diff
@@ -3,138 +3,233 @@ id: feedback-service
...
+}
+```
+#### Delete feedback
+
+- Operation Name - DeleteFeedback
+- Variables - __id_
+
+Example Mutation
+
+```bash
```
Comment (hybridx) [inline packages/documentation-spa/docs/microservices/feedback/feedback-service.md:211]:
```suggestion
```graphql
```

File: packages/documentation-spa/docs/microservices/feedback/feedback-service.md [hunk 6]
```diff
@@ -3,138 +3,233 @@ id: feedback-service
...
+```
+
+
+#### Update feedback search index
+
+- Operation Name - UpdateFeedbackIndex
+
+Example Mutation
+
+```bash
```
Comment (hybridx) [inline packages/documentation-spa/docs/microservices/feedback/feedback-service.md:229]:
```suggestion
```graphql
```

**Review by hybridx — APPROVED** [review]

File: packages/documentation-spa/docs/microservices/feedback/feedback-service.md (hunk 5 above)
Comment (riginoommen) [reply packages/documentation-spa/docs/microservices/feedback/feedback-service.md:211]: AFAIK there is no official markdown support for graphql

File: packages/documentation-spa/docs/microservices/feedback/feedback-service.md (hunk 6 above)
Comment (riginoommen) [reply packages/documentation-spa/docs/microservices/feedback/feedback-service.md:229]: AFAIK there is no official support for markdown in graphql

File: packages/documentation-spa/docs/microservices/feedback/feedback-service.md (hunk 4 above)
Comment (riginoommen) [reply packages/documentation-spa/docs/microservices/feedback/feedback-service.md:179]: AFAIK there is no official support for markdown in graphql

File: packages/documentation-spa/docs/microservices/feedback/feedback-service.md (hunk 3 above)
Comment (riginoommen) [reply packages/documentation-spa/docs/microservices/feedback/feedback-service.md:136]: AFAIK there is no official support for markdown in graphql

File: packages/documentation-spa/docs/microservices/feedback/feedback-service.md (hunk 2 above)
Comment (riginoommen) [reply packages/documentation-spa/docs/microservices/feedback/feedback-service.md]: Updated

**Review by ghanlohar — APPROVED** [review]
Looks good to me.

**Review by deshmukhmayur — APPROVED** [review]

**Pull request merged**